core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/AutoTuner.scala

```diff
@@ -754,17 +754,32 @@ class AutoTuner(
       limitedLogicList: Option[Seq[String]] = Some(Seq("spark.sql.shuffle.partitions")),
       showOnlyUpdatedProps: Boolean = true):
       (Seq[RecommendedPropertyResult], Seq[RecommendedCommentResult]) = {
-    filterByUpdatedPropertiesEnabled = showOnlyUpdatedProps
-    limitedLogicList.foreach { limitedSeq =>
-      limitedSeq.foreach(_ => limitedLogicRecommendations.add(_))
-    }
-    skipList.foreach(skipSeq => skipSeq.foreach(_ => skippedRecommendations.add(_)))
-    if (processPropsAndCheck) {
-      initRecommendations()
-      calculateRecommendations()
+    val isOnGpu = {
+      val isPluginLoaded = getPropertyValue("spark.plugins") match {
+        case Some(f) => f.equals("com.nvidia.spark.SQLPlugin")
+        case None => false
+      }
```

Comment:
`spark.plugins` could be a list, right?
If so, then `f.equals("com.nvidia.spark.SQLPlugin")` will be always false or crash if the type does not match.